packages/near-lake-framework/README.md

```diff
@@ -0,0 +1,37 @@
+# NEAR Lake Framework
+
+NEAR Lake Framework is a syntactical framework for accessing the NEAR Lake Indexer Data. It is a wrapper around the AWS API.
+
+Has `@near-lake/primitives` as a dependency, which contains the types for the data with associated helper functions.
+
+## Simple Example
+
+```ts
+import { startStream, types } from '@near-lake/framework';
+
+const lakeConfig: types.LakeConfig = {
+    s3BucketName: 'near-lake-data-mainnet',
+    s3RegionName: 'eu-central-1',
+    startBlockHeight: 66264389,
+};
+
+async function handleStreamerMessage(
+    block: types.Block,
+    context: types.LakeContext
+): Promise<void> {
+    let events = block.eventsByAccountId("x.paras.near")
+    console.log(events)
+}
+
+(async () => {
+    await startStream(lakeConfig, handleStreamerMessage);
+})();
+```
+
+The interface to capture data about an event has the following arguments:
```

Comment:
I think the following may be unnecessary information for an indexer developer to know. This should be presented in the ReadMe for @near-lake/primitives

Reply:
Fine by me, happy to make the change later on today or tomorrow unless anybody else thinks otherwise